tests/test_connections_page.py

```diff
@@ -0,0 +1,68 @@
+import pytest
+from utilities.data_randomizer import generate_invalid_invitation_code
+
+
+@pytest.mark.smoke
+@pytest.mark.regression
+def test_update_connection_status(go_to_connections_page):
+    connections_page = go_to_connections_page
+    # activate > deactivated
+    active_status = connections_page.get_connection_status()
+    deactivated_status = connections_page.invoke_dropdown_menu().deactivate_connection().get_connection_status()
+    assert deactivated_status != active_status, 'Status is not updated'
+    # deactivated > active
+    reactivated_status = connections_page.invoke_dropdown_menu().activate_connection().get_connection_status()
+    assert reactivated_status != deactivated_status, 'Status is not updated'
+    assert reactivated_status == active_status, 'Status is not updated'
+
+
+def test_default_invitation_dialog(go_to_connections_page):
+    connections_page = go_to_connections_page
+    connections_page.is_accept_code_dialog_shown()
+    # check content
+    expected_description = 'Please paste the invitation code, received from your partner'
+    expected_textarea_label = 'Enter the invitation code'
+    actual_description = connections_page.get_description()
+    actual_textarea_label = connections_page.get_textarea_label()
+    assert actual_description == expected_description, 'Incorrect modal dialog description'
+    assert actual_textarea_label == expected_textarea_label, 'Incorrect text area label'
+    connections_page.close_invitation_dialog()
+
+
+@pytest.mark.regression
+def test_invalid_invitation_code(go_to_connections_page):
+    connections_page = go_to_connections_page
+    connections_page.is_accept_code_dialog_shown()
```

Comment:
same here

Reply:
^ the same update of invitation code dialog was applied.